opentelemetry-api/src/opentelemetry/trace/span.py

```diff
@@ -232,6 +245,9 @@ def is_recording_events(self) -> bool:
     def end(self, end_time: typing.Optional[int] = None) -> None:
         pass
 
+    def get_attribute(self, key: str) -> types.AttributeValue:
+        pass
+
     def set_attribute(self, key: str, value: types.AttributeValue) -> None:
         pass
```

Comment:
This sounds like a job for `property` :superhero:

Reply:
Can we use a property decorator here. As the interface requires a setter and getter method.

Reply:
From the [Python docs](https://docs.python.org/3/library/abc.html#abc.abstractproperty):

```py
class C(ABC):
    @property
    def x(self):
        ...

    @x.setter
    @abstractmethod
    def x(self, val):
        ...
```